package.json

```diff
@@ -11,6 +11,7 @@
     "react-dom": "^16.5.2",
     "react-jss": "^8.6.1",
     "react-lottie": "^1.2.3",
+    "react-modal": "^3.8.1",
```

Comment:
is there a reason we're pulling in react-modal instead of just using grommet?

Reply:
i was having issues getting grommet's layer component to work because of errors that were coming from the grommet library so i pulled in that component instead